pkgs/applications/audio/cardinal/default.nix

```diff
@@ -48,6 +59,7 @@ stdenv.mkDerivation rec {
     libarchive
     liblo
     libsamplerate
+    libsndfile
     mesa
     python3
```

Comment:
python3 is nativeBuildInput as well: the build fails for me with 
```
cardinal> /nix/store/561wgc73s0x1250hrgp7jm22hhv7yfln-bash-5.2-p15/bin/bash: line 1: python3: command not found
```

Reply:
I figured mesa is one as well.

Reply:
mesa is a library, so that sounds suspicious. Maybe try building without mesa altogether...